ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/QueryAnonymizerTest.java

```diff
@@ -0,0 +1,89 @@
+package io.confluent.ksql.test;
+
+import io.confluent.ksql.test.QueryTranslationTest.QttTestFile;
+import io.confluent.ksql.test.loader.JsonTestLoader;
+import io.confluent.ksql.test.tools.TestCase;
+import io.confluent.ksql.util.GrammarTokenExporter;
+import java.io.FileReader;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import org.approvaltests.Approvals;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+import io.confluent.ksql.util.QueryAnonymizer;
+import org.junit.Before;
+import org.junit.Test;
+
+public class QueryAnonymizerTest {
+
+  private static final Path QUERIES_TO_ANONYMIZE_PATH =
+      Paths.get("src/test/java/io/confluent/ksql/test/QueriesToAnonymizeTest.txt");
+  private static final Path QUERY_VALIDATION_TEST_DIR = Paths.get("query-validation-tests");
+  private final QueryAnonymizer anon = new QueryAnonymizer();
+  private final Stream<TestCase> testCases = testFileLoader().load();
+  private List<String> sqlTokens;
+
+  @Before
+  public void setUp() {
+    sqlTokens = GrammarTokenExporter.getTokens();
+    sqlTokens.addAll(Arrays.asList("", "INT", "DOUBLE", "VARCHAR", "BOOLEAN", "BIGINT", "VALUE", "KEY"));
```

Comment:
Also add lowercase version of all tokens. Why are you adding empty string as a token?

Reply:
The anonymizer returns all tokens capitalized so there isn't a need for the lower case versions. I removed the empty string.